woocommerce/woocommerce-ios
# Shipping Labels Onboarding: first CTA screen

Closes #6915 

### Description
As part of the Shipping Labels Onboarding project, we should implement the first screen which is the entry point of the flow after tapping the banner in the order detail screen.
The CTA screen is dismissable, after pressing the `Not now` button.

### Testing instructions
1. Launch the app, and select a website without the WCShip extension installed (you can create a Jurassic ninja website, like a did). The feature flag `shippingLabelsOnboardingM1` should be enabled (enabled by default in debug mode). You can also add `return true` under `OrderDetailsDataSource` -> `shouldAllowWCShipInstallation` property.
2. Open the Order List.
3. Select an order.
4. You should be able to see the onboarding banner of WCShip after the products list.
5. By tapping on the banner, you should see the new CTA screen.
6. Tap the button "What is WooCommerce Shipping?" -> You should see the web view displaying what is WCShip.
7. Tap the button "Not now" -> The view should be dismissed.

### Screenshots
<img src="https://user-images.githubusercontent.com/495617/175551674-a33322b7-12d3-4425-8d22-8aca20b538f7.png" width=300 />



---
